app/controllers/katello/api/v2/hosts_bulk_actions_controller.rb

```diff
@@ -314,7 +314,9 @@ def find_errata
     end
 
     def find_host_collections
-      @host_collections = HostCollection.where(:id => params[:host_collection_ids])
+      throw_each_resource_not_found(name: 'host collection', expected_ids: params[:host_collection_ids]) do
```

Comment:
@jlsherrill thoughts on using the block pattern for `throw_resource_not_found` ?

Reply:
whats the alternative? Passing the expected_ids as parameter ?